visualisation/volscope.cpp

```diff
@@ -0,0 +1,187 @@
+/**
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ **/
+/**
+ * @file volscope.cpp
+ * @ingroup visualization
+ * @author David Coeurjolly (\c david.coeurjolly@cnrs.fr )
+ * Laboratoire d'InfoRmatique en Image et Systèmes d'information - LIRIS (CNRS, UMR 5205), CNRS,
+ * France
+ *
+ * @date 2023/12/22
+ *
+ * Vol visualization using polyscope.
+ *
+ * This file is part of the DGtal library.
+ */
+#include <vector>
```

Comment:
```suggestion
#define NO_ADD_STBIMAGE_IMPLEMENT //To avoid duplicated linking errors (like LNK2005 in MSVC)
#include <vector>
```
It should fix the MSVC error as from the contrib PR ;)